addons/website_sale_loyalty/views/website_sale_templates.xml

```diff
@@ -146,8 +146,7 @@
     </template>
 
     <template id="reduction_coupon_code" inherit_id="website_sale.reduction_code">
-        <xpath expr="//t[@t-set='force_coupon']" position="replace">
-            <t t-set='force_coupon' t-value="website_sale_order.pricelist_id.code or request.params.get('code_not_available') or website_sale_order.get_promo_code_error(delete=False)"/>
```

Comment:
Is it also necessary to drop the `get_promo_code_error` call ?
As it's the only place where it's called, I fear we are losing the support of an existing logic here :thinking:
If that's what we want, we should remove the logic fully in the fw-port PR in master.

Reply:
If I leave it, then it also not work properly. From what I see it's also called in line 20 and 22 to show the error message.